end-to-end-tests-v2/package.json

```diff
@@ -7,31 +7,33 @@
     "simulateMetaTxn:local": "source ./.env.local && PRIVATE_KEY=$PRIVATE_KEY && FORWARDER_ADDRESS=$FORWARDER_ADDRESS && WORLD_ADDRESS=$WORLD_ADDRESS && forge script script/SimulateMetaTxn.s.sol:SimulateMetaTxn --fork-url $RPC_URL --private-key $PRIVATE_KEY --broadcast --sig \"run(address)\" $WORLD_ADDRESS -vvvv",
     "transferERC20:local": "source ./.env.local && PRIVATE_KEY=$PRIVATE_KEY && WORLD_ADDRESS=$WORLD_ADDRESS && forge script script/TransferERC20.s.sol:TransferERC20 --fork-url $RPC_URL --private-key $PRIVATE_KEY --broadcast --sig \"run(address)\" $WORLD_ADDRESS -vvvv",
     "lint": "pnpm run prettier && pnpm run solhint",
     "prettier": "prettier '**/*.sol' --plugin=prettier-plugin-solidity --check",
     "fixFormat": "prettier '**/*.sol' --plugin=prettier-plugin-solidity --write",
-    "solhint": "solhint --config ./.solhint.json 'src/**/*.sol'"
+    "solhint": "solhint --config ./.solhint.json 'src/**/*.sol'",
+    "test": "chmod +x run-scripts.sh && ./run-scripts.sh"
```

Comment:
Can we use this to run all tests via a run-test.sh script? The script could:
1.  Deploy and configure the world
2.  Run all the Forge scripts in sequence
3.  Add some helpful logs
Then the test command can just call this script, which keeps package.json cleaner